### PR TITLE
Add possibility to change ttl on open DB

### PR DESCRIPTION
**Pull request by falkevik**
We have seen cases where it could be good to change TTL on already open DB.
Change ttl in TtlCompactionFilterFactory on open db.
Next time a filter is created, it will filter accroding to the set TTL.

Is this something that could be useful for others?
Any downsides?

### CODE REVIEW

**Comment by miasantreble** [comment]
@falkevik thanks for contributing to RocksDB.
Do you mind adding a simple test to verify the TTL is indeed updated?

File: utilities/ttl/db_ttl_impl.cc [hunk 1]
```diff
@@ -309,5 +309,15 @@ Iterator* DBWithTTLImpl::NewIterator(const ReadOptions& opts,
   return new TtlIterator(db_->NewIterator(opts, column_family));
 }
 
+void DBWithTTLImpl::SetTtl(ColumnFamilyHandle *h, int32_t ttl) {
+    std::shared_ptr<TtlCompactionFilterFactory> filter;
+    Options opts;
+    opts = GetOptions(h);
+    filter = std::static_pointer_cast<TtlCompactionFilterFactory>(opts.compaction_filter_factory);
```
Comment (miasantreble) [inline utilities/ttl/db_ttl_impl.cc]: line too long, please limit width to 80 characters.

Comment (falkevik) [reply utilities/ttl/db_ttl_impl.cc]: ok, will fix that.

**Comment by falkevik** [comment]
ok, will add a test.

**Pull request closed without merging**